internal/cloud/backend.go

```diff
@@ -599,12 +616,22 @@ func (b *Cloud) StateMgr(name string) (statemgr.Full, error) {
 	return &remote.State{Client: client}, nil
 }
 
+// Return the WorkspaceOutputs from TFC
+func (b *Cloud) WorkspaceOutputs(name string) ([]*tfe.WorkspaceOutputs, error) {
+	w, err := b.WorkspaceWithOutputs(name)
+	if err != nil {
+		return nil, fmt.Errorf("Failed to retrieve workspace %s information: %v", name, err)
```

Comment:
💅 Good practice when using fmt.Errorf to use %w to wrap the underlying error. I know this isn't done everywhere but nice to know about https://go.dev/blog/go1.13-errors